src/Security/AdherentLoginTimestampRecorder.php

```diff
@@ -28,8 +28,8 @@ public function onSecurityInteractiveLogin(InteractiveLoginEvent $event)
             return;
         }
 
-        if (!$token instanceof UsernamePasswordToken) {
-            throw new \RuntimeException(sprintf('Authentication token must be a %s instance.', UsernamePasswordToken::class));
+        if (!$token instanceof TokenInterface) {
```

Comment:
This test is basically useless now as `$event` is always returning a `TokenInterface`.
Let's remove it no ?